cookbook/deployment/heroku.rst

```diff
@@ -132,6 +132,18 @@ variables, you can issue a single command to prepare your app for a deployment:
 
     $ heroku config:set SYMFONY_ENV=prod
 
+.. caution::
+
+    Be aware that dependencies from ``composer.json`` listed in ``require-dev``
+    are never installed during a deploy on Heroku. This may cause problems if your
+    Symfony environment relies on such packages. For instance, the ``dev``
+    environment of the Symfony Standard Edition loads the SensioGeneratorBundle
```

Comment:
I think it is usefull to have a caution about this, but I think we have to make it a bit shorter. What do you think about:

```rst
.. caution::

    Be aware that dependencies from ``composer.json`` listed in the ``require-dev``
    section are never installed during a deploy on Heroku. This may cause problems if
    your Symfony environment relies on such packages. The solution is to move this
    packages from ``require-dev`` to the ``require`` section.
```